packages/medusa/package.json

```diff
@@ -53,7 +53,7 @@
     "@types/lodash": "^4.14.191",
     "awilix": "^8.0.0",
     "body-parser": "^1.19.0",
-    "bull": "^3.12.1",
+    "bullmq": "^3.5.6",
```

Comment:
question: The core does not depends on bullmq right?

Reply:
Yes, the job scheduler still depends on it.